### PR TITLE
[followup] Local GraphQL schema

### PR DESCRIPTION
**Pull request by dimitropoulos**
This is a followup from https://github.com/Kong/insomnia/pull/3803. Closes INS-814.

Each commit links to the thing it followed up on in the commit description.

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.tsx [hunk 1]
```diff
@@ -342,33 +337,42 @@ class GraphQLEditor extends PureComponent<Props, State> {
...
       const content = JSON.parse(file.toString());
       if (!content.data) {
         throw new Error('JSON file should have a data field with the introspection results');
       }
 
-      newState.schema = buildClientSchema(content.data);
-      newState.schemaLastFetchTime = Date.now();
+      if (this._isMounted) {
+        return;
+      }
```
Comment (develohpanda) [inline packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.tsx]: Shouldn't this be

```suggestion
      if (!this._isMounted) {
        return;
      }
```

**Comment by develohpanda** [comment]
Were we not planning on putting in dropdown-dividers between the docs/remote/local options?

File: packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.tsx (hunk 1 above)
Comment (dimitropoulos) [reply packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.tsx]: um... YES!!

fixed in 2b8af1cd781f3e8cc377d637cc18c19d10b1ac97

**Comment by dimitropoulos** [comment]
ahh, right, right, dividers added in https://github.com/Kong/insomnia/pull/3847/commits/41acf622471e99798a01ee2550d777a8229dd005, but I'm not 100% remembering if this is what we talked about.

Screenshot_20210727_142910

**Review by develohpanda — APPROVED** [review]
LGTM! Both the error and success states look to be working. The dividers are what we discussed as well (noted in the linear ticket) 👍🏽

**Review by gatzjames — APPROVED** [review]
From a UX pov I also feel that figuring out which schema is being used is a bit confusing (local/remote) imo.
Other than that I couldn't find anything else except a bug in the variables editor.
LGTM!

**Pull request merged**